packages/vkui/src/components/AdaptivityProvider/AdaptivityProvider.tsx

```diff
@@ -19,7 +22,24 @@ export const AdaptivityProvider = ({
   hasHover,
   children,
 }: AdaptivityProviderProps) => {
-  const bridge = useBridgeAdaptivity();
+  // TODO [>=6]: удалить использование хука (#5049)
+  /* eslint-disable @typescript-eslint/naming-convention */
+  const LEGACY_isPerhapsPropsByBridgeTypeAdaptive =
+    viewWidth !== undefined && viewHeight !== undefined;
+  const LEGACY_isPerhapsPropsByBridgeTypeForceMobile =
+    viewWidth !== undefined && sizeX !== undefined && sizeY !== undefined;
+  const LEGACY_disableInternalUseBridgeAdaptivity =
+    LEGACY_isPerhapsPropsByBridgeTypeAdaptive || LEGACY_isPerhapsPropsByBridgeTypeForceMobile;
+  const LEGACY_bridge = useBridgeAdaptivity(LEGACY_disableInternalUseBridgeAdaptivity);
+  /* eslint-enable @typescript-eslint/naming-convention */
+
+  if (process.env.NODE_ENVIRONMENT === 'development') {
+    // TODO [>=6]: удалить warn
+    if (!LEGACY_disableInternalUseBridgeAdaptivity) {
+      warn("[@vkontakte/vk-bridge's deprecated] Если вы используете VK Bridge, то используйте хук `useAdaptivity()` из @vkontakte/vk-bridge-react и результат передайте в компонент (см. https://github.com/VKCOM/VKUI/issues/5049)"); // prettier-ignore
```

Comment:
```suggestion
      warn("[@vkontakte/vk-bridge] Интеграция VKUI с @vkontakte/vk-bridge устарела и будет удалена в v6. Используйте хук useAdaptivity() из @vkontakte/vk-bridge-react и результат передайте в компонент (см. https://github.com/VKCOM/VKUI/issues/5049)"); // prettier-ignore
```
Сам бридж-то не депрекейтед, только наша интеграция с ним, сообщение вводит в заблуждение мощно.

Reply:
❤️ ✅  https://github.com/VKCOM/VKUI/pull/5496/commits/e31e6ac4207ac9275a2defc6e3b1f65b303d0c75